src/main/java/sirius/db/mixing/BaseEntity.java

```diff
@@ -180,6 +180,24 @@ private void throwEmptyFieldError(Mapping field, Property property) {
                         .handle();
     }
 
+    /**
+     * Asserts that the given field remains unchanged.
+     * <p>
+     * This can be used for conditions where a field cannot be updated after creation.
+     *
+     * @param field the field to check
+     */
+    public void assertFieldChanged(Mapping field) {
```

Comment:
The method name does not describe the functionality, maybe something like `assertFieldUnchanged`?

Reply:
Maybe also `Mapping...` instead of `Mapping` to allow more use cases? But would need some small adaptions when generating the message

Reply:
Dachte auch. Aber der `InvalidFieldException` erwartet genau ein Feld, sonst brauchen wir eine andere Exception.

Reply:
kk